src/components/navigation/Breadcrumbs/BreadcrumbAction.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { colors } from '../../theme';
+
+class BreadcrumbAction extends React.Component {
+  state = { hovering: false };
+
+  render() {
+    const hover = () => {
+      this.setState({ hovering: true });
+    };
+    const unhover = () => {
+      this.setState({ hovering: false });
+    };

```

Comment:
Is tracking `onHover` necessary? If we use `styled-components` we can keep this simple with a css selector.